src/equations/serre_green_naghdi_1d.jl

```diff
@@ -423,7 +833,15 @@ is a conserved quantity (for periodic boundary conditions).
 
 For a [`bathymetry_flat`](@ref) the total energy is given by
 ```math
-\\frac{1}{2} g h^2 + \\frac{1}{2} h v^2 + \\frac{1}{6} h^3 v_x^2.
+\\frac{1}{2} g \\eta^2 + \\frac{1}{2} h v^2 + \\frac{1}{6} h^3 v_x^2.
+```
+For a [`bathymetry_mild_slope`](@ref) the total energy is given by
```

Comment:
```suggestion
For a [`bathymetry_mild_slope`](@ref) the total modified energy is given by
```
The same is also a few lines above (where I cannot comment).